tests/unit/anchore_engine/analyzers/test_utils.py

```diff
@@ -40,3 +40,15 @@ def test_missing_value_with_fail(self, mixed_data):
     def test_missing_index_with_fail(self, mixed_data):
         with pytest.raises(IndexError):
             assert dig(mixed_data, "a", "d", 111, fail=True)
+
+    def test_none_value(self):
+        assert dig({"a": None}, "a") == None
+
+    def test_empty_value_with_force_default(self):
+        assert dig({"a": ""}, "a", force_default=12) == 12
+
+    def test_false_value_with_force_default(self):
+        assert dig({"a": False}, "a", force_default=12) == 12
+
+    def test_explicit_empty_value_with_force_default(self):
+        assert dig({"a": None}, "a", force_default=12) == 12
```

Comment:
Nit: It might be nice to have a dig test that specifies `force_default` when that value _shouldn't_ be used (i.e. when a truthy value is found in the object) just for completeness

Reply:
done!